.github/workflows/template_gitops.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         uses: actions/checkout@v3
 
       - name: GitOps (build, push and deploy a new Docker image)
-        uses: Staffbase/gitops-github-action@v4
+        uses: Staffbase/gitops-github-action@v4.1
```

Comment:
Isn't `v4.1` included in `v4`? Or did I miss something?

Reply:
The tag was not updated so far as I know. 😄